content/en/product_analytics/charts/funnel_analysis.md

```diff
@@ -76,31 +75,79 @@ To combine an event, click the three dots next to an event and select **+ Combin
+**Note**: The `Group by` option is not supported for the funnel steps visualization; when applied, the visualization automatically changes to a top list.
 
 ## Refine conversion
 
 You can further analyze the information on the funnel page to understand the conversion rate. Conversion rate is a crucial metric that measures the effectiveness of your site or application.
 
-You can analyze conversion by **session count** or **users**, which means you can understand how many sessions or users ended up completing the funnel.
+You can analyze conversion by **session**, **users** or **accounts**. This can be useful if you suspect, for instance, that a minority of your user base converts at a high rate.
+
+- If you select <strong>Session</strong>, all steps must be completed within the same <code>@session.id</code> to count as a conversion.
+
+- If you select <strong>User</strong>, the funnel requires the same individual user (<code>@user.id</code>) to complete every step for the conversion to count.
+
+- If you select <strong>Account</strong>, different users within the same account can complete different steps and the conversion still counts. In this case, the funnel is tied to the <code>@account.id</code> facet.
+
+{{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_refine_conversion.png" alt="Specify which facet you'd like to use to refine your conversion. The available options are Session, User, and Account." style="width:50%;" >}}
+ 
+
 
-You can measure conversion by the following attributes:
 
-- **Conversion count** - A count of users who went through the funnel you've defined.
-- **Conversion rate** - A conversion refers to the moment when a user responds to a call to action. This rate is the percentage of users who have entered the funnel and converted.
+You can measure conversion by the following attributes:
 
-  **Note**: Only completed RUM sessions contribute to conversion rates.
+- **Conversion count**: A count of users who went through the funnel you've defined.
+- **Conversion rate**: A conversion refers to the moment when a user responds to a call to action. This rate is the percentage of users who have entered the funnel and converted.
+- **Time to convert**: The time it took for the user to complete the step events. This is not available for the _Steps_ visualization, it changes to timeseries when selected.
 
-- **Time to convert** - The time it took for the user to complete the step events.
 
 You can measure these attributes **across all steps** or between **specific steps**.
 
 {{< img src="product_analytics/journeys/funnel_analysis/pana_funnel_conversion.png" alt="Measure attributes across all steps or specific steps." style="width:60%;" >}}
 
-Use the **filter** selector to filter by various criteria that you define.
+Use the **filter** selector to scope to the various criteria you want to define. These filters are applied to all steps in your funnel.
```

Comment:
Does this refer to a different UI from the "Filtering" section? If so, I would explain it further. If not, I would either take this out, or link to that section. What we want to avoid here is confusing your user by making them wonder if they've been able to follow the way you've described the UI - we don't want them to have to mentally jump around to figure that out 🙂